core/src/main/java/org/apache/calcite/avatica/Meta.java

```diff
@@ -647,11 +648,12 @@ public static CursorFactory create(@JsonProperty("style") Style style,
     public static final CursorFactory LIST =
         new CursorFactory(Style.LIST, null, null, null);
 
-    public static CursorFactory record(Class resultClazz) {
-      return new CursorFactory(Style.RECORD, resultClazz, null, null);
+    public static CursorFactory record(Class resultClass, List<Field> fields,
+                                       List<String> fieldNames) {
+      return record(Style.RECORD_PROJECTION, resultClass, fields, fieldNames);
```

Comment:
Fix the indentation of line 652, `List<String> fieldNames) {`. Then it will be clear in the diff that this method's signature has not changed.

Reply:
It's done, sorry to have missed that.